addons/mail/static/src/core/common/message.xml

```diff
@@ -53,7 +53,7 @@
                                     (from <a t-att-href="message.resUrl"><t t-esc="message.originThread?.prefix"/><t t-esc="message.originThread?.displayName"/></a>)
                                 </t>
                             </small>
-                            <div t-if="props.message.scheduledDate" t-att-class="{ 'ms-2': props.isInChatWindow and isAlignedRight }" t-att-title="messageService.scheduledDateSimple(props.message)">
+                            <div t-if="props.message.scheduledDate" t-att-class="{ 'ms-2': props.isInChatWindow and isAlignedRight or (!isOriginThread and !message.is_transient) }" t-att-title="messageService.scheduledDateSimple(props.message)">
```

Comment:
Also I would add parenthesis on the left part of the `or` too as I'm always confused with priority (also to be consistent as you add them on the right part).